docs/fernet.rst

```diff
@@ -35,12 +35,15 @@ has support for implementing key rotation via :class:`MultiFernet`.
         they'll also be able forge arbitrary messages that will be
         authenticated and decrypted.
 
-    .. method:: encrypt(data)
+    .. method:: encrypt(data, current_time=None)
```

Comment:
I think we'd rather have an `encrypt_at_time()` API, instead of an optional parameter, as a way of encouraging people not to use it except in test :-)